test/test_storage.bats

```diff
@@ -0,0 +1,106 @@
+setup() {
+    load 'lib/bats-support/load'
+    load 'lib/bats-assert/load'
+    load 'lib/common.bash'
+}
+
+teardown() {
+    ${KUBECTL} delete pod -l app=storage-test -n "$TARGET_NAMESPACE" --as system:admin
+    ${KUBECTL} delete pvc -l app=storage-test -n "$TARGET_NAMESPACE" --as system:admin
+}
+
+@test "Test Storage with One PVC per Node" {
+    # Get all worker nodes
+    nodes=$(${KUBECTL} get nodes --no-headers -l node-role.kubernetes.io/worker -o custom-columns=":metadata.name")
+
+    # Create PVCs for each node
+    for node in $nodes; do
+        cat <<EOF | ${KUBECTL} apply -n "$TARGET_NAMESPACE" --as system:admin -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: storage-test-pvc-$node
+  labels:
+    app: storage-test
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: ocs-external-storagecluster-ceph-rbd
+EOF
+    done
+
+    # Create Pods for each node
+    for node in $nodes; do
+        cat <<EOF | ${KUBECTL} apply -n "$TARGET_NAMESPACE" --as system:admin -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: storage-test-pod-$node
+  labels:
+    app: storage-test
+spec:
+  tolerations:
+    - key: "nvidia.com/gpu.product"
+      operator: "Equal"
+      value: "NVIDIA-A100-SXM4-40GB"
+      effect: "NoSchedule"
+    - key: "nvidia.com/gpu.product"
+      operator: "Equal"
+      value: "Tesla-V100-PCIE-32GB"
+      effect: "NoSchedule"
+  containers:
+  - name: test-container
+    image: quay.io/libpod/alpine
```

Comment:
Please use our custom test image for all pods:

```suggestion
    image: ghcr.io/ocp-on-nerc/openshift-tests:latest
```

This is built automatically from the `image` directory, so it's easy for us to add any necessary tooling.